### DISCUSSION TITLE
General comments from the documentation website

### DISCUSSION THREAD

**Discussion by Diego635** [jaminzoda/terra-antiqua#237] [General]
### To simplify the manual and documentation website, we have decided not to have comments on the pages but rather here. This is a general discussion thread, feel free to comment on anything about Terra Antiqua.

**Comment by lppbear** [comment]
when I deal with setting paleoshoreline into a paleoDEM, I run the set paleoshoreline utility, then an error show up as could not delete a tif file generated in the Temp file of windows system. This error reads "deleting c:\users\lin_c\appdata\local\temp\rasterized_vector_layer.tiff failed: permission denied". I use the QGIS v3.4 and a window 10 system.
